src/sentry/static/sentry/app/icons/iconSentryFull.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import SvgIcon from './svgIcon';
+
+type Props = React.ComponentProps<typeof SvgIcon>;
+
+const IconSentryFull = React.forwardRef(function IconSentryFull(
+  props: Props,
+  ref: React.Ref<SVGSVGElement>
+) {
+  return (
+    <svg viewBox="0 0 601 134" width="1em" height="1em" {...props} ref={ref}>
```

Comment:
Should the width and height be equal if the image is a rectangle? The old dimensions were 601px wide 134px tall.